src/pretix/control/templates/pretixcontrol/organizers/detail.html

```diff
@@ -7,7 +7,7 @@
         {% blocktrans with name=organizer.name %}Organizer: {{ name }}{% endblocktrans %}
         <a href="{% url "control:organizer.edit" organizer=organizer.slug %}"
            class="btn btn-default">
-            <span class="fa fa-edit"></span>
+            <span class="fa fa-sliders"></span>
```

Comment:
Sorry I didn't raise a flag before changing this. My bad!

Bringing in organizer `Display settings` into the general organizer's edit page; I was thinking of an icon that would intuitively tell the user that he/she can do more that just general edits. Something `settlings-like`. However, now that I know why it must remain that way, I'll roll this back. :)